### ISSUE TITLE
Explain settings options

### ISSUE THREAD

**Issue by BasThomas** [rnystrom/GitHawk#323]
Most options in Settings are pretty hard to understand without more context. For example, what would "Read on Open" or "Badge Unread" mean to new users?

We should look into how we can enable that. iOS does this by adding a footer to the cell, but seems to only do this once per section.

![img_1412](https://user-images.githubusercontent.com/4190298/30587773-ab3ff1a4-9d34-11e7-9115-b302a7d95949.PNG)

![screen shot 2017-09-18 at 9 06 02 pm](https://user-images.githubusercontent.com/12063704/30550689-d7611fb2-9cb5-11e7-99b8-bac474ccc36a.png)

**Comment by Sherlouk** [comment]
May be an idea to break it up a bit more than it is now as well?

You've got "Report a Bug", "View Source Code", "Star on Github" which are sort of GitHawk related but more so actions

Then "Badge Unread" which I feel could fall under it's own _refresh_ title with an option to turn off background refresh entirely (without having to use app settings, ideally), badge unread and badge mentions (#292).

Also "Read on Open" and "GitHawk Signature" are generic settings which definitely need more explaining on their own

**Comment by rizwankce** [comment]
+1 for explanations for settings options.

iOS has the settings explanations on lots of places inside **Settings** app. For example, take a look at Settings -> General -> Accessibility 

Note that if a settings option needs explanations, IOS has separate Section header title, cell and footer title for it.

AFAIK there is no option to enable it. However from popular form builder Projects like [Eureka](https://github.com/xmartlabs/Eureka#section-header-and-footer) and [XLForm](https://github.com/xmartlabs/XLForm) having Separate section is the only easy way.

So we can group options under a section (w/ header title) and whichever settings need explanations we can have a separate section (w/ footer title)

**Comment by BasThomas** [comment]
Yep, think you'd have to do so manually, but that should be quite easy. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Huddie** [comment]
I think breaking it up by: Github/ GitHawk / Personal Settings would be a good thing now. We just added a push notifications switch and the 2nd section is a bit packed. I'd suggest breaking it up with: 

GitHawk

- Report a Bug
- View Source Code
- Write a Review

Personal

- Double Tap Reaction
- Badge Unread
- Push Notifications
- Read on Open
- GitHawk Signature
